.circleci/config.yml

```diff
@@ -57,10 +57,10 @@ jobs:
             pip install --progress-bar off .[checking]
 
       - run:
-          name: autopep8
+          name: black
           command: |
             . venv/bin/activate
-            autopep8 . -r --diff --exit-code
+            black . --line-length 99 --exclude="docs" --check
```

Comment:
@hvy Problem of using `setup.cfg` is that black does not support `setup.cfg`... They only support `project.toml` c.f. https://github.com/psf/black/issues/683#issuecomment-542731068 .

The difference of `setup.cfg` and `project.toml` are:

- `setup.cfg`
    - file type: .ini
    - they are supporting mypy, flake8, but not black.
- `project.toml`
    - file type: .toml
    - they are supporting black, but not mypy or flake8.

So, we can not support everything by `setup.cfg` or `project.toml` 😢 

Solutions:
1. Keep `setup.cfg` and add `project.toml`.
2. Run black with options s.t. `black . --line-length 99 --exclude="docs"`.

Reply:
Thanks for the clarification, I wasn't aware of the fact that `setup.cfg` wasn't supported. Let's proceed with 2 (i.e. no further changes regarding this are required). I think it might be worth considering embracing PEP 517, supporting `pyproject.toml` in the future.